gossip/store_certificates_test.go

```diff
@@ -5,10 +5,13 @@ import (
 
 	"github.com/0xsoniclabs/sonic/scc"
 	"github.com/0xsoniclabs/sonic/scc/cert"
+	scc_node "github.com/0xsoniclabs/sonic/scc/node"
 	"github.com/Fantom-foundation/lachesis-base/inter/idx"
 	"github.com/stretchr/testify/require"
 )
 
+var _ scc_node.Store = (*Store)(nil)
```

Comment:
why is this dependency needed ?

Reply:
In the near future the `gossip.Store` is supposed to serve as the `node.Store` for the one `Node` instance managing the SCC. Until this is integrated, there is no implicit check that makes sure that the `gossip.Store` is implementing this interface. With this line I add this requirement explicitly on the `gossip.Store` implementation.